hw/ip/flash_ctrl/dv/cov/flash_ctrl_cov.vRefine

```diff
@@ -0,0 +1,59 @@
+    </ccf-files>
+  </information>
+  <rules>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/tck_i" entityType="toggle" excTime="1652803318" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/tdi_i" entityType="toggle" excTime="1652803318" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/tms_i" entityType="toggle" excTime="1652803318" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/tdo_o" entityType="toggle" excTime="1652803318" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/bist_enable_i" entityType="toggle" excTime="1652803988" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/scanmode_i" entityType="toggle" excTime="1652803988" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/scan_en_i" entityType="toggle" excTime="1652803988" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/scan_rst_ni" entityType="toggle" excTime="1652803988" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/flash_power_ready_h_i" entityType="toggle" excTime="1652803988" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/flash_power_down_h_i" entityType="toggle" excTime="1652803988" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/flash_test_mode_a_io" entityType="toggle" excTime="1652803988" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/flash_test_voltage_h_io" entityType="toggle" excTime="1652803988" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/flash_err_o" entityType="toggle" excTime="1652803988" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
```

Comment:
i think for flash error it makes sense to manually force an error during a flash operation (read / program / erase) and check the corresponding error status is registered in flash. Did you guys plan for this in V2S? sorry i cannot remember.

Reply:
All these signals were provided by you in #12325 to be excluded. We did not plan that in V2S.

Reply:
sounds good.